src/main/resources/modules/acute_myeloid_leukemia.json

```diff
@@ -0,0 +1,806 @@
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 409089005,
+          "display": "Febrile neutropenia (disorder)"
+        }
+      ],
+      "direct_transition": "Check_Bacteremia"
+    },
+    "Non-Febrile_Neutropenia": {
+      "type": "ConditionOnset",
+      "assign_to_attribute": "",
+      "target_encounter": "Chemotherapy_Inpatient_Encounter",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 47318007,
+          "display": "Neutropenia (disorder)"
+        }
+      ],
+      "direct_transition": "Transfer_to_Stepdown"
+    },
+    "Bacteremia": {
+      "type": "ConditionOnset",
+      "assign_to_attribute": "bacteremia",
+      "target_encounter": "Chemotherapy_Inpatient_Encounter",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 5758002,
+          "display": "Bacteremia (finding)"
+        }
+      ],
+      "distributed_transition": [
+        {
+          "transition": "Transfer_to_ICU",
+          "distribution": 0.112
+        },
+        {
+          "transition": "Transfer_to_Stepdown",
+          "distribution": 0.8620000000000001
+        },
+        {
+          "transition": "Death_Event",
+          "distribution": 0.026
+        }
+      ]
+    },
+    "Check_Bacteremia": {
+      "type": "Simple",
+      "complex_transition": [
+        {
+          "condition": {
+            "condition_type": "Active Medication",
+            "codes": [
+              {
+                "system": "RxNorm",
+                "code": 199885,
+                "display": "levofloxacin 500 MG Oral Tablet"
+              }
+            ]
+          },
+          "distributions": [
+            {
+              "transition": "Bacteremia",
+              "distribution": 0.137
+            },
+            {
+              "transition": "No_Bacteremia",
+              "distribution": 0.863
+            }
+          ]
+        },
+        {
+          "condition": {
+            "condition_type": "Attribute",
+            "attribute": "levofloxacin",
+            "operator": "is nil"
+          },
+          "distributions": [
+            {
+              "transition": "Bacteremia",
+              "distribution": 0.33
+            },
+            {
+              "transition": "No_Bacteremia",
+              "distribution": 0.6699999999999999
+            }
+          ]
+        }
+      ]
+    },
+    "No_Bacteremia": {
+      "type": "Simple",
+      "distributed_transition": [
+        {
+          "transition": "Transfer_to_ICU",
+          "distribution": 0.041
+        },
+        {
+          "transition": "Death_Event",
+          "distribution": 0.015
+        },
+        {
+          "transition": "Transfer_to_Stepdown",
+          "distribution": 0.944
+        }
+      ]
+    },
+    "Death": {
+      "type": "Death",
+      "exact": {
+        "quantity": 1,
+        "unit": "days"
+      },
```

Comment:
Not a big deal, but did you want this 1 day delay before Death to be here? 

You can click "Remove Exact/Range" in the module builder if you did not want this here.